woocommerce/woocommerce-android
# [TTP] Add successful refund notification after TTP test payment

Closes: #8843


### Description

The PR adds a snack bar with an action that indicates that the refund was done automatically.  A user can open the order details screen too

Also, it moves the refund code to the TTP summary screen and adds snack bar that indicates that refund is in progress

### Testing instructions

Test the cases:
* Refund successful and navigate to the order details screen via snack bar action
* Refund not successful and then it opens order details automatically and notifies a user with snack bar (you may do this by modifying condition on `TapToPaySummaryViewModel::101`
* Notice that text on the TTP summary screen is changed to indicate that refund will be done automatically


### Images/gif


https://user-images.githubusercontent.com/4923871/233287865-dc7f9ab1-dede-4f8c-9dcc-4e13bca94842.mp4
